lib/ethereumex/counter.ex

```diff
@@ -41,7 +41,6 @@ defmodule Ethereumex.Counter do
     :ets.update_counter(@tab, key, {2, count}, {key, 0})
   end
 
-  @spec do_increment(binary() | nil, atom(), integer()) :: integer()
```

Comment:
why did you remove this spec?

Reply:
because it's here already: https://github.com/mana-ethereum/ethereumex/pull/65/commits/35ff360b35541fbe5e7ce39236b3d6e8baa9bc02#diff-2917486c2b82f1420bd9a6a7bbaeb602L39